CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 ### Added
 - support for custom attributes based on new database columns
 
+### Fixed
+- properly catch errors from shopware basked when validating items for check stock
```

Comment:
Please check the typing! Also there is no need for an empty line between change log sections

Reply:
I removed the empty line and improved the entry wording.